eng/pipelines/cadl-ci.yaml

```diff
@@ -125,3 +125,8 @@ jobs:
           jdkArchitectureOption: 'x64'
           publishJUnitResults: false
 
+      - script: |
+          node node_modules/\@azure-tools/cadl-ranch/dist/cli/cli.js check-coverage node_modules/@azure-tools/cadl-ranch-specs/http || echo "cadl ranch tests not fully covered"
```

Comment:
Add this `|| echo "cadl ranch tests not fully covered"` to avoid build failure when cadl  tests are not fully covered.